101-event-grid-cloudevents/azuredeploy.json

```diff
@@ -0,0 +1,189 @@
+        "description": "Name of Event Grid Subscription"
+      }
+    },
+    "eventGridSubscriptionDeliverySchema": {
+      "type": "string",
+      "metadata": {
+        "description": "Event Grid Subscription delivery schema"
+      },
+      "allowedValues": [
+        "EventGridSchema",
+        "CloudEventV01Schema"
+      ],
+      "defaultValue": "CloudEventV01Schema"
+    },
+    "eventGridSubscriptionEndpointType": {
+      "type": "string",
+      "metadata": {
+        "description": "Event Grid Subscription endpoint type"
+      },
+      "allowedValues": [
+        "Webhook"
+      ],
+      "defaultValue": "Webhook"
+    },
+    "eventGridSubscriptionIncludedEventTypes": {
+      "type": "string",
+      "metadata": {
+        "description": "Comma delimited list of filters for Event Grid Subscription"
+      },
+      "defaultValue": "All"
+    },
+    "eventGridSubscriptionIsSubjectCaseSensitive": {
+      "type": "bool",
+      "metadata": {
+        "description": "Value indicating whether the filter subjectBeginsWith and subjectEndsWith are case sensitive or not"
+      },
+      "defaultValue": false
+    }
+  },
+  "variables": {
+    "logicApp": {
+      "name": "[parameters('logicAppName')]",
+      "apiVersion": "2017-07-01",
+      "resourceId": "[resourceId('Microsoft.Logic/workflows', parameters('logicAppName'))]",
+      "location": "[parameters('location')]",
+      "triggerId": "[resourceId('Microsoft.Logic/workflows/triggers', parameters('logicAppName'), 'manual')]"
+    },
+    "eventGridTopic": {
+      "name": "[parameters('eventGridTopicName')]",
+      "apiVersion": "2018-05-01-preview",
+      "resourceId": "[resourceId('Microsoft.EventGrid/topics', parameters('eventGridTopicName'))]",
+      "location": "[parameters('location')]",
+      "schema": "[parameters('eventGridTopicSchema')]"
+    },
+    "eventGridSubscription": {
+      "name": "[parameters('eventGridSubscriptionName')]",
+      "apiVersion": "2018-05-01-preview",
+      "location": "[parameters('location')]",
+      "deliverySchema": "[parameters('eventGridSubscriptionDeliverySchema')]",
+      "endpoint": {
+        "type": "[parameters('eventGridSubscriptionEndpointType')]"
+      },
+      "filter": {
+        "includedEventTypes": "[split(parameters('eventGridSubscriptionIncludedEventTypes'), ',')]",
+        "isSubjectCaseSensitive": "[parameters('eventGridSubscriptionIsSubjectCaseSensitive')]"
+      }
+    },
+    "tags": {
+      "author": "Justin Yoo",
+      "profile": "https://twitter.com/justinchronicle",
+      "projectUrl": "https://github.com/Azure/azure-quickstart-templates",
+      "repositoryUrl": "https://github.com/Azure/azure-quickstart-templates",
+      "license": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/LICENSE"
+    }
+  },
+  "resources": [
+    {
+      "comments": "### RESOURCE - LOGIC APP - EVENT HANDLER ###",
+      "apiVersion": "[variables('logicApp').apiVersion]",
```

Comment:
Use a literal apiVersion so schema validation can be done.

Reply:
It passes validation.